polaris/polaris/models.py

```diff
@@ -305,9 +349,33 @@ class Transaction(models.Model):
     * **error**
     
         catch-all for any error not enumerated above.
+        
+    SEP-31 Statuses:
+    
+    * **pending_sender**
+    
+        awaiting payment to be initiated by sending anchor.
```

Comment:
Should the whitespace be under this line instead of above?

Reply:
It looks like it in the code but when its rendered, this is right.